Update README.md (#169)
aiven/http-connector-for-apache-kafka: README.md

- batching;
- delivery retries.

> An example of configuration (file connect-http-sink.json):

Reviewer: ```suggestion
An example of configuration (file `connect-http-sink.json`):
```

}
```

> Then deploy the connector configuration against the Kafka Connect REST API with the following command: 

Reviewer: ```suggestion
Then deploy the connector configuration against the Kafka Connect REST API with the following command:
```

}
```

> Then execute the config json against the Apache Kafka REST API `curl -d @config/connect-http-sink.json -H "Content-Type: application/json" -X POST http://localhost:8083/connector`

Reviewer: ```suggestion
Then deploy the connector configuration against the Kafka Connect REST API with the following command: 

```shell
curl -d @config/connect-http-sink.json -H "Content-Type: application/json" -X POST http://localhost:8083/connector
```

```
